src/igs_network.c

```diff
@@ -4120,16 +4133,23 @@ void igsagent_set_name (igsagent_t *agent, const char *name)
               strlen (previous) + strlen ("-IGS-CALLS") + 1);
             snprintf (previous_calls_channel, IGS_MAX_AGENT_NAME_LENGTH + 15,
                       "%s-IGS-CALLS", previous);
+            char *previous_igs_channel = (char *) zmalloc (
+              strlen (previous) + strlen ("-IGS") + 1);
+            snprintf (previous_igs_channel, IGS_MAX_AGENT_NAME_LENGTH + 15,
```

Comment:
Use `strlen()` instead of magic numbers to avoid these kinds of mistakes.  
You probably wanted to write `IGS_MAX_AGENT_NAME_LENGTH + 5` instead of `IGS_MAX_AGENT_NAME_LENGTH + 15`